install_config/certificate_customization.adoc

```diff
@@ -18,6 +18,25 @@ xref:../architecture/infrastructure_components/web_console.adoc#architecture-inf
 This can be done during a
 xref:../install/configuring_inventory_file.adoc#advanced-install-custom-certificates[cluster installation] or configured after installation.
 
+[[using-certificate-chains]]
+== Configuring a Certificate Chain
+If a certificate chain is used, then all certificates must be manually
+concatenated into a single named certificate file. These certificates must
+be placed in the following order:
+
+* {product-title} master host certificate
+* Intermediate CA certificate
+* Root CA certificate
+* Third party certificate
+
+To create this certificate chain, concatenate the certificates into a
+common file. You must run this command for each certificate and ensure
+that they are in the order specified above.
```

Comment:
If you swap out "above," this sounds great.
> ! above adj
Do not use to indicate a relative location in a document, as in “the above restrictions.” Use “previous” or
“preceding.”

Reply:
This sentence now ends in, "...previously defined order."

Thanks for catching this!

Reply:
Beautiful. Thank you for making the change!